modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -51,20 +49,23 @@ You can create more than one API specification in a single project. If you want
 As you draft, the text editor suggests RAML nodes, methods, and other elements that you can add at the location of your cursor.
 +
 You can import files into your project, either separately or bundled together in .zip files. The files can be RAML 1.0 files, JSON files, or OpenAPI Specification (OAS) 2.0 files. The files can be on your computer or you can specify a URL for them if they are located online.
-+
-//For more information about importing files and steps for how to import them, see "Import Files into an API Project", which is linked to from the *See also* section at the end of this topic.
-
 
 
 == What to Do Next
 
-You can publish the API fragment to Anypoint Exchange. Click the *Publish* button that is in the top-right corner of the text editor, and then click *Publish to Exchange* to open the *Publish API specification to Exchange* dialog.
+You can publish the API fragment to Anypoint Exchange. Click the *Publish* button that is in the top-right corner of the text editor, and then click *Publish to Exchange*. Specify a version number for the project, which Exchange treats as an _asset_. This number is displayed in Exchange in the *Version* section of the right-hand pane in the page for the project.
+
+.The asset version of your API-fragment project appears in *Versions* section in the right pane of the asset page.
+image:apid-location-of-asset-version-in-exchange.png[The location of the Versions section]
```

Comment:
It just occurred to me that if we are providing an image that shows WHERE the GUI item is located, then why do we need to describe the location in words in the narrative?